### PR TITLE
try to help reproducing #3103

### PR DESCRIPTION
**Pull request by forki**

### CODE REVIEW

File: src/Paket.Core/Dependencies/PackageResolver.fs [hunk 1]
```diff
@@ -1264,10 +1264,7 @@ let Resolve (getVersionsRaw : PackageVersionsFunc, getPreferredVersionsRaw : Pre
                             UnlistedSearch = true
                         }
                     else
-                        { flags with 
-                            Ready = true 
-                            UnlistedSearch = true 
-                        }
+                        flags
```
Comment (forki) [inline src/Paket.Core/Dependencies/PackageResolver.fs]: /cc @cloudRoutine @matthid 

I'm scared!

Comment (matthid) [reply src/Paket.Core/Dependencies/PackageResolver.fs]: You even added a test-case for this scenario. What could go wrong if everything is green ;)

Comment (matthid) [reply src/Paket.Core/Dependencies/PackageResolver.fs]: Do we have test-cases for "unlisted" scenarios?

Comment (forki) [reply src/Paket.Core/Dependencies/PackageResolver.fs]: I think there are integration tests.

**Comment by forki** [comment]
@matthid ok tests seem to run into endless loop now. So I guess this is the wrong place. I guess there is something wrong at the call site. Will take a deeper look on Monday

File: src/Paket.Core/Dependencies/PackageResolver.fs [hunk 2]
```diff
@@ -1093,34 +1093,17 @@ let Resolve (getVersionsRaw : PackageVersionsFunc, getPreferredVersionsRaw : Pre
...
-        let matchingStep =
-            let currentNames =
-                conflicts
-                |> Seq.map (fun c ->
-                    c.Graph
-                    |> Set.map (fun (pr:PackageRequirement) -> pr.Name) 
-                    |> Set.add c.Name)
-                |> Set.unionMany
-
-            priorConflictSteps
```
Comment (forki) [inline src/Paket.Core/Dependencies/PackageResolver.fs]: @mrinaldi IIRC this was a shortcut that you introduced. I would like to reintroduce it again, but I don't know how.

**Pull request merged**